packages/esm-hiv-app/src/api/api.ts

```diff
@@ -1,34 +1,4 @@
 /* eslint-disable no-debugger, no-console */
 import { openmrsFetch } from '@openmrs/esm-framework';
-import { finalHIVCodeConcept, finalPositiveHIVValueConcept, computedHIV_StatusConcept } from '../constants';
 
 const BASE_WS_API_URL = '/ws/rest/v1/';
-
-export function fetchPatientsFinalHIVStatus(patientUUID: string) {
-  return openmrsFetch(
-    `/ws/fhir2/R4/Observation?code=${finalHIVCodeConcept}&value-concept=${finalPositiveHIVValueConcept}&patient=${patientUUID}&_sort=-date&_count=1`,
-  ).then(({ data }) => {
-    if (data.entry?.length) {
-      return data.entry[0].resource.valueCodeableConcept.coding[0].display;
-    }
-    return '';
-  });
-}
-
-export function fetchPatientComputedConcept_HIV_Status(patientUUID: string) {
```

Comment:
@pirupius these function were not being used on the `hiv-app` they are used on the `patientHivStatus.ts` and this file lives on the `commons-lib`. I would be out of the scope of this PR to make these changes... I just deleted the functions because they were not being used.